nix/sources.json

```diff
@@ -1,4 +1,16 @@
 {
+    "doctest-parallel": {
+        "branch": "main",
+        "description": "Test interactive Haskell examples",
+        "homepage": null,
```

Comment:
I think you should change this from `null` to `"https://github.com/martijnbastiaan/doctest-parallel"`

Reply:
Ah I just used `niv add`. So my guess is that this is some cabal field I missed in `doctest-parallel`

Reply:
Which is to say, I'm going to ignore it until a new `doctest-parallel` release :smiley_cat: